aws/data_source_aws_secretsmanager_secret_version.go

```diff
@@ -20,7 +21,7 @@ func dataSourceAwsSecretsManagerSecretVersion() *schema.Resource {
 				ForceNew: true,
 			},
 			"secret_string": {
-				Type:      schema.TypeString,
```

Comment:
Changing an attribute type is a breaking change for existing Terraform configurations. This new behavior should be handled by creating a new attribute and a new acceptance test added to cover if and when this new attribute is available.

There are also other potential issues with introducing this second attribute as briefly outlined in: https://github.com/terraform-providers/terraform-provider-aws/pull/4613#issuecomment-391337108